apps/cli/src/index.ts

```diff
@@ -60,4 +60,4 @@ program
 if (process.argv.length < 3) {
   program.help();
 }
-program.parse();
+program.parse(process.argv);
```

Comment:
We must explicitly provide `process.argv` to ensure CommandsJS process arguments correctly when running the CLI from electron